src/context/sessionContext/__snapshots__/index.test.tsx.snap

```diff
@@ -23,3 +23,27 @@ exports[`Testing SessionContext matches snapshot matches snapshot 1`] = `
   </div>
 </body>
 `;
+
+exports[`Testing SessionContext matches snapshot matches snapshot without optional sessionId 1`] = `
```

Comment:
Haha this is a very enthusiastic matching of the snapshot :P

Reply:
ha, whoops.